test/kokoro/xds_k8s.cfg

```diff
@@ -0,0 +1,13 @@
+# Config file for internal CI
+
+# Location of the continuous shell script in repository.
+build_file: "grpc-go/test/kokoro/xds-k8s.sh"
+timeout_mins: 120
+
+action {
+  define_artifacts {
+    regex: "artifacts/**/*sponge_log.xml"
+    regex: "artifacts/**/*sponge_log.log"
```

Comment:
These don't look like regexes at all; they look like globs.  (?)

Reply:
This build configuration corresponds to an internal proto describing the build configuration and these paths are supposed to comply to this format: http://ant.apache.org/manual/Types/fileset.html

Reply:
Do we own this proto?  Is it possible to change the name of this field (or migrate to a new field with a different name)?  These aren't regexes, so naming the field `regex` is just wrong.

Reply:
No, this is owned by Kokoro.